images/dockerregistry/Dockerfile

```diff
@@ -4,23 +4,4 @@
 #
 # The standard name for this image is openshift/origin-docker-registry
 #
-FROM openshift/origin-base
-
-RUN INSTALL_PKGS="origin-dockerregistry" && \
-    yum --enablerepo=origin-local-release install -y ${INSTALL_PKGS} && \
-    rpm -V ${INSTALL_PKGS} && \
-    yum clean all
-
-COPY config.yml ${REGISTRY_CONFIGURATION_PATH}
-
-LABEL io.k8s.display-name="OpenShift Container Platform Image Registry" \
-      io.k8s.description="This is a component of OpenShift Container Platform and exposes a Docker registry that is integrated with the cluster for authentication and management." \
-      io.openshift.tags="openshift,docker,registry"
-
-# The registry doesn't require a root user.
-USER 1001
-EXPOSE 5000
-VOLUME /registry
-ENV REGISTRY_CONFIGURATION_PATH=/config.yml
-
-CMD /usr/bin/dockerregistry ${REGISTRY_CONFIGURATION_PATH}
+FROM openshift/origin-docker-registry:latest
```

Comment:
@soltysh @liggitt @deads2k ignore this for now, just testing :-)

Reply:
@deads2k @smarterclayton @stevekuznetsov not sure if this is kosher for the rebase to land, but it seems to unblock our GCE conformance and other tests that are building the registry image.

This will just pick-up the latest image from DockerHub which I guess is OK until we figure out the proper way?

Reply:
> @deads2k @smarterclayton @stevekuznetsov not sure if this is kosher for the rebase to land, but it seems to unblock our GCE conformance and other tests that are building the registry image.
> 
> This will just pick-up the latest image from DockerHub which I guess is OK until we figure out the proper way?

If this works, I can live with it as long as don't push official images based on this docker file.

Reply:
If I land the job changes today to build this image from `openshift/image-registry` can we just nuke this?

Reply:
yes, this is just for signal for the moment

Reply:
OK, let me deliver that today

Reply:
@liggitt @deads2k @soltysh @mfojtik FYI this is delivered

Reply:
looks like master still uses this in `os::build::images()`

Reply:
As I understand it, it's this flag in the installer: https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/common/test_cases/origin_built_installed_release.yml#L78

/cc @sdodson 

We build the image in Origin, but we re-build it from the other repo and tag _that_ image as the real one today.